### PR TITLE
docs: Bootstrapping guide prose for CLI

### PR DESCRIPTION
**Pull request by wardbell**
See issue #19510.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

File: aio/content/guide/bootstrapping.md [hunk 1]
```diff
@@ -1,65 +1,83 @@
...
-Every application has at least one NgModule, the _root_ module
+An Angular Module (NgModule) class describes how the application parts fit together.
+Every application has at least one Angular Module, the _root_ module 
 that you [bootstrap](#main) to launch the application.
-You can call it anything you want. The conventional name is `AppModule`.
+You can call the class anything you want. The conventional name is `AppModule`.
 
-The [setup](guide/setup) instructions produce a new project with the following minimal `AppModule`.
-You'll evolve this module as your application grows.
+The [**Angular CLI**](https://cli.angular.io/) produce a new project with the following minimal `AppModule`.
```
Comment (Foxandxss) [inline aio/content/guide/bootstrapping.md]: wouldnt be better to point to the getting started?

**Review by Foxandxss — APPROVED** [review]

File: aio/content/guide/bootstrapping.md [hunk 2]
```diff
@@ -124,58 +128,42 @@ You can call the one _root_ component anything you want but most developers call
...
-## More about NgModules
+## More about Angular Modules
 
 Your initial app has only a single module, the _root_ module.
 As your app grows, you'll consider subdividing it into multiple "feature" modules,
 some of which can be loaded later ("lazy loaded") if and when the user chooses
 to visit those features.
 
-When you're ready to explore these possibilities, visit the [NgModules](guide/ngmodule) guide.
+When you're ready to explore these possibilities, visit the [Angular Module](guide/ngmodule) guide.
```
Comment (gkalpak) [inline aio/content/guide/bootstrapping.md]: _Angular Module_ --> _Angular Modules_

File: aio/content/guide/bootstrapping.md (hunk 1 above)
Comment (gkalpak) [inline aio/content/guide/bootstrapping.md]: _produce_ --> _produces_

File: aio/content/guide/bootstrapping.md [hunk 3]
```diff
@@ -1,65 +1,83 @@
...
 </div>
 
+{@a imports}
 
+### The _imports_ array
 
-<div class="l-sub-section">
+Angular Modules are a way to consolidate features that belong together into discrete units.
+Many features of Angular itself are organized as Angular Modules.
+HTTP services are in the `HttpModule`. The router is in the `RouterModule`.
```
Comment (gkalpak) [inline aio/content/guide/bootstrapping.md]: `HttpModule` --> `HttpClientModule`(?)

File: aio/content/guide/bootstrapping.md (hunk 1 above)
Comment (Foxandxss) [reply aio/content/guide/bootstrapping.md]: I love when you correct Ward :joy:

**Comment by vicb** [comment]
missing:
- target label
- merge label
- green CI

**Review by Foxandxss — APPROVED** [review]

**Pull request merged**